deployment/volumes/uploads.yaml

```diff
@@ -9,4 +9,4 @@
       - ReadWriteOnce
     resources:
       requests:
-        storage: {{ .Values.uploadsStorage }}
+        storage: 25Gi
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-16T07:12:28Z" title="Thursday, April 16th 2020, 9:12:28 am +02:00">Apr 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -9,4 +9,4 @@
       - ReadWriteOnce
     resources:
       requests:
-        storage: {{ .Values.uploadsStorage }}
+        storage: 25Gi
```


But why we have changed this here if we don’t merge? @mattwr18

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-16T07:32:16Z" title="Thursday, April 16th 2020, 9:32:16 am +02:00">Apr 16, 2020</time>_

---


I wasn't suggesting we shouldn't merge... I was suggesting that instead of just merging these changes, we have a quick copy of the actual values in `deployments`, `services`, `pvc`, etc. and make sure there are not other things missing.
Also, we could consider how it's organized. What _we_ actually use, and what is just for others who might want to use local file systems for uploads, for example.